src/nlp/dataset.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+import configparser
+from typing import NoReturn, Tuple
+from utils import preprocess_title
+from torch.utils.data import Dataset
+from transformers import BertTokenizer
+
+CONFIG = configparser.ConfigParser()
+CONFIG.read("nlp_config.ini")
+MODEL_NAME = CONFIG["model_configs"]["model_name"]
+MAX_LEN = CONFIG["dataset_configs"]["max_length"]
+TOKENIZER = BertTokenizer.from_pretrained(MODEL_NAME)
+
+
+class ShopeeNLPDataset(Dataset):
```

Comment:
It's better to not inherit the `torch.utils.data.Dataset` class because of the Abhishek Thakur recommendation. In this case, the only change is to return torch.tensor instead of np.ndarray